container/Containerfile

```diff
@@ -0,0 +1,237 @@
+        " && \
+    # NFS-Ganesha
+    PACKAGES="$PACKAGES \
+        dbus-daemon \
+        nfs-ganesha-ceph${GANESHA_PACKAGE_VERSION} \
+        nfs-ganesha-rados-grace${GANESHA_PACKAGE_VERSION} \
+        nfs-ganesha-rados-urls${GANESHA_PACKAGE_VERSION} \
+        nfs-ganesha-rgw${GANESHA_PACKAGE_VERSION} \
+        nfs-ganesha${GANESHA_PACKAGE_VERSION} \
+        rpcbind \
+        sssd-client \
+        " && \
+    # ISCSI
+    PACKAGES="$PACKAGES \
+        ceph-iscsi \
+        tcmu-runner \
+        python3-rtslib \
+        " && \
+    # Ceph-CSI
+    # TODO: coordinate with @Madhu-1 to have Ceph-CSI install these itself if unused by ceph
+    #       @adk3798 does cephadm use these?
+    PACKAGES="$PACKAGES \
+        attr \
+        ceph-fuse${CEPH_PACKAGE_VERSION} \
+        rbd-nbd${CEPH_PACKAGE_VERSION} \
+        " && \
+    # Rook (only if packages must be in ceph container image)
+    PACKAGES="$PACKAGES \
+        systemd-udev \
+        " && \
+    # Util packages (should be kept to only utils that are truly very useful)
+    # 'sgdisk' (from gdisk) is used in docs and scripts for clearing disks (could be a risk? @travisn @guits @ktdreyer ?)
+    # 'ps' (from procps-ng) and 'hostname' are very valuable for debugging and CI
+    # TODO: Per discussion, sg3_utils should be moved to ceph.spec.in with libstoragemgmt; @ktdreyer
+    #       ref: https://github.com/ceph/ceph-container/pull/2013#issuecomment-1248606472
+    PACKAGES="$PACKAGES \
+        gdisk \
+        hostname \
+        procps-ng \
+        sg3_utils \
+        " && \
+    echo ${PACKAGES} > packages.txt && \
+    echo "=== PACKAGES TO BE INSTALLED ===" && \
+    cat packages.txt && \
+    echo "=== INSTALLING ===" && \
+    dnf install -y --setopt=install_weak_deps=False --enablerepo=powertools $(cat packages.txt) && \
+    mkdir -p /var/run/ganesha
+
+# Disable sync with udev since the container can not contact udev
+RUN \
+    sed -i -e 's/udev_rules = 1/udev_rules = 0/' \
+           -e 's/udev_sync = 1/udev_sync = 0/' \
+           -e 's/obtain_device_list_from_udev = 1/obtain_device_list_from_udev = 0/' \
+        /etc/lvm/lvm.conf && \
+    # validate the sed command worked as expected
+    grep -sqo "udev_sync = 0" /etc/lvm/lvm.conf && \
+    grep -sqo "udev_rules = 0" /etc/lvm/lvm.conf && \
+    grep -sqo "obtain_device_list_from_udev = 0" /etc/lvm/lvm.conf
+
+# CLEAN UP!
+RUN set -x && \
+    dnf clean all && \
+    rm -rf /var/cache/dnf/* && \
+    rm -rf /var/lib/dnf/* && \
+    rm -f /var/lib/rpm/__db* && \
+    # remove unnecessary files with big impact
+    rm -rf /etc/selinux /usr/share/{doc,man,selinux} && \
+    # don't keep compiled python binaries
+    find / -xdev \( -name "*.pyc" -o -name "*.pyo" \) -delete
+
+# Verify that the packages installed haven't been accidentally cleaned, then
+# clean the package list and re-clean unnecessary RPM database files
+RUN rpm -q $(cat packages.txt) && rm -f /var/lib/rpm/__db* && rm -f *packages.txt
```

Comment:
Revising the clean up from ceph-container, I am seeing an ARM image that is 1.07GB on my machine, versus 1.25GB to 1.28GB for other `quay.io/ceph` images I've pulled.

Of note, the parens in the `find ... -delete` command fix a bug where `*.pyc` files weren't being deleted before. Plus removing unnecessary DNF and RPM database files.